core/admin/mailu/configuration.py

```diff
@@ -16,13 +16,14 @@
     'DOMAIN_REGISTRATION': False,
     'TEMPLATES_AUTO_RELOAD': True,
     'MEMORY_SESSIONS': False,
+    'FETCHMAIL_ENABLED': False,
```

Comment:
I think the default must be True to not break existing deployments not having the environment variable set.

Reply:
Yeah, that seems reasonable. I'll change te default to `True`.